### DISCUSSION TITLE
Feature: Unifi MFA workaround

### DISCUSSION THREAD

**Discussion by AStoker** [koush/scrypted#1335] [Ideas]
Unifi is going to start enabling MFA in July (https://community.ui.com/questions/Upcoming-Multi-Factor-Authentication-MFA-Requirement-for-All-UI-Accounts-in-July-2024/e0d06d3c-c0ae-4af6-aa67-53c159d486a8). As I understand it, this will break the Unifi plugin for Scrypted.  

Ideally, before July 22nd, we can have some means of enabling MFA with Unifi without breaking Scrypted.

**Comment by YouCan-doit** [comment]
Hope there is solution for this “problem”?
Because 22 july is coming:)

**Reply by AStoker** [reply]
So, it might not require anything after all (but probably need to wait to see). I think it might be more of the primary Unifi accounts that require the cloud login that will require MFA. But scrypted uses a local login for your router, and I don't believe that needs MFA.
I went ahead and experimented (at my own risk) and enabled the MFA for my primary account (the one I use to log in normally) and that did not break the setup. 🤞

**Comment by YouCan-doit** [comment]
Great thanks. I whas thinking about that to. Ill have also mfa enabled now;)Its just works. RegardsRaymondOp 26 mrt 2024 om 16:23 heeft Andrew Stoker ***@***.***> het volgende geschreven:﻿
So, it might not require anything after all (but probably need to wait to see). I think it might be more of the primary Unifi accounts that require the cloud login that will require MFA. But scrypted uses a local login for your router, and I don't believe that needs MFA.
I went ahead and experimented (at my own risk) and enabled the MFA for my primary account (the one I use to log in normally) and that did not break the setup. 🤞

**Comment by spryanca** [comment]
I enabled 2FA, which disabled my Unifi Protect camera in Scrypted.  So, I installed the RTSP plugin in Scrypted and used the RTSP feeds from Protect to bring my UniFi Protect Feed back into Scrypted.

**Comment by koush** [comment]
read the docs? use a local login.
